### PR TITLE
Add image to popular projects, Notification Handling implemented

### PR DESCRIPTION
**Pull request by svr8**
# Problem
- A mechanism to handle all bundled notifications was needed as described in issue [13](https://github.com/codeuino/Social-Platform-Donut/issues/13)
- Images had to be added before names of popular names of projects as stated in issue [91](https://github.com/codeuino/Social-Platform-Donut/issues/91)

# Solution
- Solution to # 13 : Publisher/Subscriber model makes it easy for various GUI components to interact with updates in notifications. This is implemented in `views/js/events.js`. Now, in the file `views/js/notify.js` , we subscribe to the notifications. No GUI is implemented yet. Notifications will be displayed in console.
- Solution to # 91: Regarding images before popular project names, a separated `div` with css attribute `display-inline` was added. Size of image can be changed using css class `project-img`.

# Before and After Screenshots
## Before
Before preview
## After
After preview

### **Type of Change**
- [ ] Bug fix
- [ ] New Feature
- [x] Development of UI/UX prototypes
- [ ] Small refactor
- [ ] Change in Documentation

### **Checklist**
- [x] My code follow the same style as the codebase
- [ ] My Code changed required change in documentation
- [ ] I have updated the Readme accordingly
- [x] I made PR within **development branch only**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "express": "^4.16.3",
     "mongo": "^0.1.0",
     "mongodb": "^3.1.0",
-    "mongoose": "^5.1.7",
+    "mongoose": "^5.4.17",
```
Comment (invincibleJai) [inline package.json]: any specific reason for updating mongoose?

File: views/css/other-landing.css [hunk 2]
```diff
@@ -36,6 +36,15 @@ form input.form-control{
   position: fixed;
   width: 255px;
 }
+.project-img{
+  width: 15px;
+  height: 15px;
+  display: inline-block;
+  margin-right: 5px;
+
+  background-image: url('/img/eventcd (2).jpg');
```
Comment (invincibleJai) [inline views/css/other-landing.css]: it'll be good if you can rename the image to something meaningful

File: package.json (hunk 1 above)
Comment (svr8) [reply package.json]: I did not intend to do it.
During `npm install`, I got a prompt to execute `npm audit fix`. Should I revert those changes?

File: views/css/other-landing.css (hunk 2 above)
Comment (invincibleJai) [reply views/css/other-landing.css]: @devshiva619 are there any services for it or we need to hardcode the image as of now?

**Comment by invincibleJai** [comment]
@svr8 thanks for the PR, but i don't think you assigned the issue. 

Before picking up any issue you need to comment as i can already it's being assigned to @imskr 

cc @devshiva619 @jaskirat2000

**Comment by svr8** [comment]
Oh! My bad. Sorry for any inconveniences.

File: package.json (hunk 1 above)
Comment (svr8) [reply package.json]: Okay. I'll undo the changes. But, should I close this PR as it was already assigned?

Comment (invincibleJai) [reply package.json]: i don't think, let it be there and do comment on the issue that you have fixed it. Am not sure if @imskr already working on it.

PLS Don't repeat this as we don't want multiple people to work on the same issue.

Comment (svr8) [reply package.json]: Sure, I'll keep that in mind.

File: views/css/other-landing.css (hunk 2 above)
Comment (devshiva619) [reply views/css/other-landing.css]: @invincibleJai as of now its hardcoded, the values will be populated from the backend.

**Comment by svr8** [comment]
I made a new proper PR [here](https://github.com/codeuino/Social-Platform-Donut/pull/132)

**Pull request closed without merging**